fx.js

```diff
@@ -281,6 +281,29 @@ module.exports = function start(filename, source) {
     findNext()
   })
 
+  // Scrolls to and sets cursor at first line of object
+  box.key('g', function () {
```

Comment:
g/G already implemented but in https://github.com/antonmedv/blessed/blob/master/lib/widgets/scrollablebox.js#L133-L178
As well as Ctrl+f/u/b/d

Maybe it's better to move all of them here?

Maybe it's be